### PR TITLE
Added include_attributes and exclude_attributes to GET /api/v1/executions/{id}

### PR DESCRIPTION
**Pull request by nmaludy**
Previously `GET /api/v1/executions/{id}` did not have the `include_attributes` or `exclude_attributes` query parameters, meaning it couldn't filter the output of large executions. This PR adds that ability.

@Kami are any other modifications necessary?

**TODO**
- [x] Tests

### CODE REVIEW

File: st2common/st2common/router.py [hunk 1]
```diff
@@ -669,7 +669,7 @@ def process_item(item):
                 result.append(item)
         elif isinstance(data, dict):
             # get_one response
-            result = process_item(item)
+            result = process_item(data)
```
Comment (Kami) [inline st2common/st2common/router.py]: Good catch. It looks we don't have test case for this scenario? :)

Comment (Kami) [reply st2common/st2common/router.py]: Besides that and test case for handling of those attributes it looks good to me :+1: 

I believe we already have a pattern for include and exclude attributes test cases. We might not just have one for "get one" API endpoints yet.

**Comment by nmaludy** [comment]
*Please don't merge this yet*

I found a least one bug in `st2client` where it's using `?include_attributes` already, but doesn't ask for of the attributes that it needs.

So far i've found the following is broken:
- `st2 execution get xxx`

**Comment by nmaludy** [comment]
Latest commit should address the `st2client` issues

**Comment by Kami** [comment]
It looks like some CLI tests are failing. I didn't have time to dig in yet (if it's related to this change or not).

**Comment by Kami** [comment]
I pushed a small change and I believe this should now work correctly.

**Comment by Kami** [comment]
The diff got really large and messy so I needed to rebase this on top of master and force push to get it sorted out.

**Pull request merged**